readme.md

```diff
@@ -61,12 +61,11 @@ flatpak install com.usebruno.Bruno
 
 # On Linux via Apt
 sudo mkdir -p /etc/apt/keyrings
+sudo apt-get update && sudo apt-get install gpg
+sudo gpg --list-keys
```

Comment:
I think it probably is too noisy to add a comment here for _why_ this is needed. But maybe squashing the commits and adding the necessity for this line there?

Reply:
You mean, something like that?

```bash
sudo gpg --list-keys && sudo gpg --no-default-keyring --keyring ...
```

Reply:
The command to list keys looks like a noop, but it actually does something (as discussed in the PR). This is surprising, so a comment may help.

But: a comment pointing out _why_ this command is there, _at that place_ is probably not relevant for 99% of users. So that's why I suggested squashing the commits and adding a line about this to the commit message.

Reply:
The [commit](https://github.com/usebruno/bruno/pull/1037/commits/0809d3f69821bb5ac644b3bb2b78616fc452f374), where I added this command, has message:

```
In APT instructions: Use "gpg --list-keys" to create ".gnupg" directory
```

Are you sure squashing needed?

Reply:
Ah, didn't see that one, my bad.

I think in general squashing commits from (esp smaller) feature branches is a good thing. But it depends a bit on the practices of the project.

Reply:
Alright, I've squished commits related to gpg now - you can see [commit](https://github.com/Nikolai2038/bruno/commit/b8a51a81261e78d7dfe6ae3462f0e717c7391e5f)